packages/liveblocks-react/src/__tests__/useInboxNotifications.test.tsx

```diff
@@ -368,10 +368,15 @@ describe("useInboxNotifications: error", () => {
     // The third retry should be made after 10s
     await jest.advanceTimersByTimeAsync(10_000);
     await waitFor(() => expect(getInboxNotificationsReqCount).toBe(4));
+    expect(result.current).toEqual({ isLoading: true });
 
     // The fourth retry should be made after 10s
     await jest.advanceTimersByTimeAsync(15_000);
     await waitFor(() => expect(getInboxNotificationsReqCount).toBe(5));
+    expect(result.current).toEqual({
+      isLoading: false,
+      error: expect.any(Error),
+    });
```

Comment:
@nimeshnayaju Thanks for fixing the unit tests! 🙏 I added in this extra assertion and noticed it doesn't pass. Maybe we should discuss on a call what we think the behaviour here should be, because I would expect this to transition to an error state at this point. Instead it remains in loading state. Maybe I'm missing something, or maybe the implementation isn't quite correct yet.